.dev/docker/entrypoint.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/ash
+## Ensure we are in /app
+
+cd /app
+
+## check for .env file and generate app keys if missing
+if [ -f /app/var/.env ]; then
+  echo "external vars exist"
+  rm /app/.env
+
+  ln -s /app/var/.env /app/
+else
+  echo "external vars don't exist"
+  rm /app/.env
+  touch /app/var/.env
+
+  ## manually generate a key because key generate --force fails
+  echo -e "Generating key"
+  APP_KEY=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+  echo -e "Generated app key: $APP_KEY"
+  echo -e "APP_KEY=$APP_KEY" > /app/var/.env
+
+  ln -s /app/var/.env /app/
+fi
+
+## check for DB up before starting the panel
+zshoecho "Checking database status."
```

Comment:
Is this an actual command or meant to be just `echo`?

Reply:
I may have been typing in another window and not noticed.